playbooks/ephor.yml

```diff
@@ -2,11 +2,7 @@
 - name: Workspace playbook for Ephor use-case
   hosts: localhost
   roles:
-    - irods_repo
-    - irods_icommands
     - irods_iselect
-    - irods_skel
     - myrods-sync
```

Comment:
Is myrods-sync still needed?

Reply:
Yes, iselect and myrods-sync are currently used by the data manager to transfer data to yoda. If they started working with ibridges we can remove these.